shared/pingpong/pingpong.go

```diff
@@ -871,7 +915,7 @@ func (pps *WorkerState) sendFromTo(
 		}
 
 		successCount++
-		accounts[from].addBalance(fromBalanceChange)
+		fromAcct.addBalance(fromBalanceChange)
```

Comment:
Since you're not indexing into the map anymore, does this need to set the new account state at the end?